frontend/express/public/core/user-management/templates/drawer.html

```diff
@@ -0,0 +1,173 @@
+                  <validation-provider rules="required" v-slot="v">
+                    <el-input v-model="drawerScope.editedObject.username" placeholder="Enter User Name"></el-input>
+                  </validation-provider>
+                </div>
+              </div>
+              <div class="user-management-drawer-content__inputs-area__input-wrapper">
+                <div class="user-management-drawer-content__label">
+                  <div class="text-small text-heading">E-mail</div>
+                  <div class="user-management-drawer-content__help-text"></div>
+                </div>
+                <div class="user-management-drawer-content__input">
+                  <validation-provider rules="required" v-slot="v">
+                    <el-input v-model="drawerScope.editedObject.email" placeholder="Enter E-mail"></el-input>
+                  </validation-provider>
+                </div>
+              </div>
+            </div>
+            <div class="bu-column user-management-drawer-content__profile-picture-area">
+              <div class="user-management-drawer-content__label">
+                <div class="text-small text-heading">Profile Picture</div>
+              </div>
+              <div class="user-management-drawer-content__profile-picture-area__upload-section">
+                <div class="user-management-drawer-content__profile-picture-area__upload-section__img-box"></div>
+                <div class="user-management-drawer-content__profile-picture-area__upload-section__description-box">
+                  <p class="user-management-drawer-content__profile-picture-area__upload-section__description-box--bold">Drag and drop or <span class="user-management-drawer-content__profile-picture-area__upload-section__description-box--link">browser</span> files to add picture</p>
+                  <p class="user-management-drawer-content__profile-picture-area__upload-section__size-warning">JPG, PNG and GIF files allowed. Maximum size is 5 MB.</p>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div v-if="drawerScope.editedObject.permission" class="user-management-drawer-content__bottom-section">
+            <div class="user-management-drawer-content__drawer-sub-section">
+              <div class="user-management-drawer-content__section-title">
+                <div>
+                  <span class="user-management-drawer-content__section-header">ROLE</span>
+                </div>
+                <div>
+                  <span class="user-management-drawer-content__help-text">
+                    Some short description can be placed here.
+                  </span>
+                </div>
+              </div>
+              <div class="user-management-drawer-content__section-content user-management-drawer-content__role-section">
+                <cly-check skin="tick" v-model="drawerScope.editedObject.global_admin" v-bind:label="'Global Administrator'"></cly-check>
```

Comment:
We deprecated <cly-check>, except the skin="star" case. You might consider migrating to element UI. Also you could pass string literals as below

`label="Global Administrator"`

instead of

`v-bind:label="'Global Administrator'" `